### PR TITLE
docs: update etiquette

### PR DESCRIPTION
**Pull request by khurrammaqbool**
### This PR:

Added: Container etiquette to docs.

### Review and tests: 
- [x] Tests pass
- [ ] Code review
- [x] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -13,6 +13,7 @@ Added:
 * Support singularity bind paths with different destination directories https://github/Clinical-Genomics/BALSAMIC/pull/1211
 * Added `--rerun-trigger mtime` option to Snakemake command https://github.com/Clinical-Genomics/BALSAMIC/pull/1217
 * `CADD` container https://github.com/Clinical-Genomics/BALSAMIC/pull/1222
+* `Container ettiquette` to ReadtheDocs https://github.com/Clinical-Genomics/BALSAMIC/pull/11
```
Comment (ivadym) [inline CHANGELOG.rst:16]:
```suggestion
* `Container ettiquette` to ReadtheDocs https://github.com/Clinical-Genomics/BALSAMIC/pull/1132
```

**Comment by khurrammaqbool** [comment]
Screenshot 2023-08-17 at 17 32 37

File: docs/bioinfo_softwares.rst [hunk 2]
```diff
@@ -74,13 +74,13 @@ manta
 multiqc
 ~~~~~~~
 :Source code: `GitHub` `<https://github.com/ewels/MultiQC>`_
-:Article: `Bioinformatics` `<https://doi.org/10.1093/bioinformatics/btw354>`_
+:Article: `Bioinformatics` `<http://doi.org/10.1093/bioinformatics/btw354>`_
```
Comment (ivadym) [inline docs/bioinfo_softwares.rst:77]: Any reasons why we should use http and not https?
```suggestion
:Article: `Bioinformatics` `<https://doi.org/10.1093/bioinformatics/btw354>`_
```

File: docs/index.rst [hunk 3]
```diff
@@ -33,8 +33,9 @@
     :hidden:
     :maxdepth: 1
 
+    README
     git_etiquette
     snakemake_etiquette
-    README
+    container_etiquette
```
Comment (ivadym) [inline docs/index.rst:39]: Probably we should group them in one document, but it's ok for now

File: docs/resources.rst [hunk 4]
```diff
@@ -25,10 +25,16 @@ References and other resources
...
 #. **VarSome**: VarSome is a knowledge base and aggregator for human genomic variants. https://varsome.com/about/
+#. **CADD**\ : CADD is a tool for scoring the deleteriousness of single nucleotide variants as well as insertion/deletions variants in the human genome. CADD can quantitatively prioritize functional, deleterious, and disease causal variants across a wide range of functional categories, effect sizes and genetic architectures and can be used prioritize causal variation in both research and clinical settings.
 
 Sample datasets
 ---------------
 
+#. **TCRB**\ : The Texas Cancer Research Biobank (TCRB) was created to bridge the gap between doctors and scientific researchers to improve the prevention, diagnosis and treatment of cancer. This work occurred with funding from the Cancer Prevention & Research Institute of Texas (CPRIT) from 2010-2014. http://txcrb.org/data.html Article: https://www.nature.com/articles/sdata201610
+=======
+Sample datasets
+---------------
```
Comment (ivadym) [inline docs/resources.rst:36]: Unresolved conflict
```suggestion
```

File: docs/requirements.txt
Comment (ivadym) [inline docs/requirements.txt:1]: It kind of works, but there are some conflicts here 🤔 

Screenshot 2023-08-17 at 17 53 23

**Review by ivadym — COMMENTED** [review]
Looks good ⭐ , but I got some conflicts when installing it locally... Have you looked into this, could we get rid of this conflict?

File: docs/requirements.txt
Comment (khurrammaqbool) [reply docs/requirements.txt:1]: Fixed

File: docs/programming_guide.rst
Comment (ivadym) [inline docs/programming_guide.rst:1]: Nice ⭐

File: docs/index.rst (hunk 3 above)
Comment (khurrammaqbool) [reply docs/index.rst]: These are grouped now.

**Review by ivadym — APPROVED** [review]
👍 💯

**Pull request merged**